proto/substrait/algebra.proto

```diff
@@ -338,6 +338,8 @@ message SetRel {
     SET_OP_INTERSECTION_MULTISET = 4;
     SET_OP_UNION_DISTINCT = 5;
     SET_OP_UNION_ALL = 6;
+    SET_OP_MINUS_DISTINCT = 7;
+    SET_OP_INTERSECTION_DISTINCT = 8;
```

Comment:
to me, the changes here and below are the opposite of what we were saying.

If default is deduplicating, then you should be introducing MINUS ALL, not MINUS DISTINCT, etc.

Reply:
Defining it how @vbarua suggested makes most sense to me too I was just trying to minimize the effect of my changes because I also initially interpreted `Minus (Primary)` to include duplicates according to the current spec.

Reply:
I'm 👍  for treating `SET_OP_MINUS_PRIMARY`, `SET_OP_MINUS_MULTISET`, `SET_OP_INTERSECTION_PRIMARY` and `SET_OP_INTERSECTION_MULTISET` as deduplicating that makes more sense although it's longer